src/workbookManager.ts

```diff
@@ -85,21 +85,39 @@ export class WorkbookManager {
     }
 
     private async updateSingleQueryAttributes(zip: JSZip, queryName: string, refreshOnOpen: boolean) {
-        const refreshOnLoadValue = refreshOnOpen ? "1" : "0";
-        const connectionId = await this.editConnections(zip, queryName, refreshOnLoadValue);
-        const sharedStringId = (await this.editSharedStrings(zip, queryName)).toString();
-        await this.editWorksheet(zip, sharedStringId);
-        await this.editPivotTable(zip, queryName, refreshOnLoadValue, connectionId);  
-    }
-
-    private async editConnections(zip: JSZip, queryName: string, refreshOnLoadValue: string) {
+        //Update connections
         const connectionsXmlString = await zip.file(connectionsXmlPath)?.async("text");
         if (connectionsXmlString === undefined) {
             throw new Error("Connections were not found in template");
+        }  
+        
+        const {connectionId, connectionString} = await this.updateConnections(connectionsXmlString, queryName, refreshOnOpen);
+        zip.file(connectionsXmlPath, connectionString);
```

Comment:
nit - connectionXmlFileString and not connectionString, just since connectionString is already a thing in this context.